spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -644,41 +644,51 @@ In instances where a Jakarta Data provider for NoSQL databases encounters a recu
+For each entity class within your application, developers can access a dedicated metamodel class, commonly called a 'Static Metamodel' class. This metamodel class follows a well-defined set of conventions, ensuring consistency and coherence in your data access layer.
 
-===== Application Requirements for a Metamodel Class
+The Static Metamodel classes can be produced through various methods:
+
+Firstly, Jakarta Data supports the generation of Static Metamodel classes using any generation tools at build time, such as Java Annotation Processor. This automatic generation occurs during compilation, offering a hassle-free way to obtain type-safe attribute access.
+
+Alternatively, application developers can craft these metamodel classes manually, tailoring them to their specific requirements. This method provides greater control and customization when working with your entity attributes.
 
-For each entity class for which the application wishes to access the metamodel,
+Moreover, Jakarta Data allows dynamic access to the metamodel through the 'jakarta.data.metamodel.Metamodel' interface. This dynamic approach is particularly beneficial when dealing with scenarios where the metamodel classes have not been generated or defined beforehand, enabling applications to adapt and interact with entity attributes on the fly.
+
+Jakarta Data's Static Metamodel empowers developers with multiple options to access entity attributes safely, whether through automatic generation, manual creation, or dynamic exploration via the 'jakarta.data.metamodel.Metamodel' interface.
+
+===== Application Requirements for a Metamodel Class
 
-- The application defines a class (the metamodel class) and annotates it with the `@StaticMetamodel` annotation.
-- The application specifies the `value` of the `@StaticMetamodel` annotation to be an entity class that the application uses in a repository as the result type of a find method or the parameter type of an insert, update, save, or delete method.
+This specification introduces the concept of a canonical metamodel, which entails the structure and conventions for creating metamodel classes.
 
-For each field of the metamodel class,
+- For every entity class "X" located within package "p," a corresponding metamodel class "X_" within package "p" is automatically generated.
+- The name of the metamodel class is derived from the entity class by appending an underscore " _ " to the entity class's name.
+- The metamodel class "X_" is required to be annotated with the `jakarta.persistence.StaticMetamodel` annotation.
```

Comment:
Is this copied from the Jakarta Persistence specification?  It refers to their annotation, `jakarta.persistence.StaticMetamodel`, rather than ours.